cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -512,6 +512,7 @@ def create_cephCSIKeyring_RBDNode(self):
         cmd_json = {"prefix": "auth get-or-create",
                     "entity": "client.csi-rbd-node",
                     "caps": ["mon", "profile rbd",
+                             "mgr", "allow rw",
```

Comment:
@aruniiird can you suggest what to change as well when the `--upgrade` flag is passed? Thanks

Reply:
@leseb this script has the support to update the caps internally if the user already exists with different cap?

Reply:
Yes, it does, that's why I'm asking @aruniiird who wrote the update logic recently to advise.

Reply:
@leseb , @Madhu-1 , in order to run upgrade you require an existing user and if you have multiple pool-prefixes you need to specify that as well. So the command may look some what like below,

python3 create-external-cluster-resources.py --upgrade --run-as-user | -u <user-name>  [ --rgw-pool-prefix <pool-prefix> ]

Most of the time you don't have to provide the `<pool-prefix>` as the script will try to internally figure out the pool prefix name.

Reply:
@aruniiird to be more precise do `create-external-cluster-resources.py` is capable of updating the user caps if the user already exists?

Reply:
currently, the script is not having any option to update the existing user caps created https://github.com/rook/rook/issues/6792 to track python changes.